src/Components/App/App.js

```diff
@@ -1,11 +1,26 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import './App.css';
 import JoinForm from '../JoinForm/JoinForm.js';
 import RoomForm from '../RoomForm/RoomForm.js';
 import { Route } from 'react-router-dom';
-import {ReactComponent as ChatLogo} from './assets/message.svg'
+import {ReactComponent as ChatLogo} from './assets/message.svg';
+import { getCircleCode } from './../../Actions/index.js';
```

Comment:
is there better syntax for the path? looks kinda gross no offense

Reply:
Not that I know of.